README.md

```diff
@@ -178,6 +178,33 @@ settings = {
 See [examples/nvim/init.lua](https://github.com/mrjosh/helm-ls/blob/master/examples/nvim/init.lua) for an
 complete example, which also includes yaml-language-server.
 
+#### Neovim + CoC plugin
+
+Using the [coc.nvim](https://github.com/neoclide/coc.nvim) plugin, you don't need the previous configuration in lua. Coc will handle the configuration instead.
```

Comment:
> you don't need the previous configuration in lua

Maybe we should restructure the readme to something like:

### Neovim config
explain filetypes etc (anything that must be done for both nvim-lspconfig and coc)

#### nvim-lspconfig
explain setup with nvim-lspconfig
#### coc
explain setup with coc

Reply:
any update on this?